doc/source/flexibility/improved_memory_engine.rst

```diff
@@ -61,6 +61,11 @@ Minimum Block Size
 
 The block size has to be big enough to store all fixed-length information in the first block. If not, the ``CREATE TABLE`` or ``ALTER TABLE`` statements will fail (see below).
 
+Limitations
+===========
+
+|MyISAM| tables are still used for temporary tables where the ``MEMORY`` tables with dynamic row format could be used instead: for schemas containing large ``VARCHAR``s, ``BLOB``, and ``TEXT`` columns.
```

Comment:
MyISAM tables are still used for query optimiser internal temporary tables where the MEMORY tables could be used now instead: for temporary tables containing large ...